app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    comm, filename, path = command.split()
+
+    if comm != "mv":
+        return
+
+    dirs = "/".join(path.split("/")[:-1])
+
+    if dirs:
```

Comment:
hey, there is a great `os.path.dirname()` method that helps to extract all folders from the path.
In addition, you can use `os.path.exists(dirs)` method to check if such a path already exists.
So as a result, you can simplify your code by entering an `if comm == "mv":` statement and implementing all your actions there.

**P.S.:** if what hit me in tg)